README.md

```diff
@@ -334,39 +241,41 @@ apt::sources:
 
-`$ puppet parser validate tests/puppetlabs-apt.pp`
+####apt::setting
 
-If you receive no output from that command, it means nothing is wrong. Then, apply the code:
+* `priority`: Integer or zero-padded integer setting the file priority. Defaults to 50.
+* `ensure`: Whether to add or remove the file. Valid values are 'present', 'absent', and 'file'. Defaults to file.
+* `source`: The source for the file. Exactly one of `content` and `source` must be specified.
+* `content`: The content for the file. Exactly one of `content` and `source` must be specified.
+* `notify_update`: Boolean for whether or not this `apt::setting` should notify `Class['apt::update']`. Defaults to true.
 
-```
-$ puppet apply --verbose tests/puppetlabs-apt.pp
-notice: /Stage[main]//Apt::Source[puppetlabs]/File[puppetlabs.list]/ensure: defined content as '{md5}3be1da4923fb910f1102a233b77e982e'
-info: /Stage[main]//Apt::Source[puppetlabs]/File[puppetlabs.list]: Scheduling refresh of Exec[puppetlabs apt update]
-notice: /Stage[main]//Apt::Source[puppetlabs]/Exec[puppetlabs apt update]: Triggered 'refresh' from 1 events>
-```    
+####apt::source
 
-The above example uses a smoke test to lay out a resource declaration and apply it on your system. In production, you might want to declare your Apt sources inside the classes where they’re needed.
+* `comment`: Add a comment to the apt source file.
+* `ensure`: Allows you to remove the apt source file. Can be 'present' or 'absent'.
+* `location`: The URL of the apt repository. Defaults to undef. Required unless `ensure => 'absent'`.
+* `release`: The distribution of the apt repository. Defaults to fact 'lsbdistcodename'.
+* `repos`: The component of the apt repository. This defaults to 'main'.
+* `include`: Hash to configure include options. Valid keys are:
+  * 'deb': References a Debian distribution's binary package.
+  * 'src': Enable the deb-src type, references a Debian distribution's source code in the same form as the include_deb type. A deb-src line is required to fetch source indexes.
+* `key`: Add key from source. Takes either a string or a hash. If a string, the value will be passed to `id` in the `apt::key`. If a hash, valid keys are:
+  * 'id': See `id` in `apt::key`. Required if a hash is specified.
+  * 'server': See `server` in `apt::key`
+  * 'content': See `content` in `apt::key`
+  * 'source': See `source` in `apt::key`
+  * 'options': See `options` in `apt::key`
+* `pin`: See apt::pin. Defaults to false.
+* `architecture`: can be used to specify for which architectures information should be downloaded. If this option is not set all architectures defined by the APT::Architectures option will be downloaded. Defaults to 'undef' which means all. Example values can be 'i386' or 'i386,alpha,powerpc'.
+* `allow_unsigned`: can be set to indicate that packages from this source are always authenticated even if the Release file is not signed or the signature can't be checked. Defaults to false. Can be 'true' or 'false'.
```

Comment:
Careful with the quotes here `'true'` and `'false'` are strings, we need it set to `true` or `false` the Booleans.